index.html

```diff
@@ -0,0 +1,179 @@
+    <link href="https://fonts.googleapis.com/icon?family=Material+Icons" rel="stylesheet">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>  
+    <link href="https://fonts.googleapis.com/css2?family=Lato&display=swap" rel="stylesheet">
+    <link href="http://fonts.cdnfonts.com/css/cocogoose" rel="stylesheet">
+  </head>
+  <body>
+      <header>
+          <nav class="header-nav">
+            <i id="menu" class="fas fa-bars hamburger-icon fa-3x burger-menu menu"></i>
+            <div class="mobile-menu">
+              <i class="fas fa-window-close close-item"></i>              
+              <div class="menu-div">
+                <a href="about.html" class="about-menu">About</a>
+                <a href="#program" class="porgram-menu">Program</a>
+                <a href="#speak" class="speak-menu">Speakers</a>
+                <a href="#partners" class="partners-menu">Partners</a>
+                <a href="#footer" class="contact-menu">Contact</a>
+                <a href="/" class="conference-menu">Conference</a>
+              </div>
+            </div>
+            <div class="header-first-line">
+              <ul class="header-list1">
+                <li><a href="/"><i class="fab fa-facebook-f"></i></a></li>
+                <li><a href="/"><i class="fab fa-twitter"></i></a></li>
+                <li><a href="/">English</a></li>
+                <li><a href="/">My Page</a></li>
+                <li><a href="/">Logout</a></li>
+              </ul> 
+            </div>
+            <div class="header-second-line">
+              <div class="header-logo">
+                <a class="header-logo-a" href="index.html">
+                  <i class="fab fa-confluence fa-2x"></i>
+                  <h2>Conference</h2>
+                </a>
+              </div>
+                <ul class="header-list2">
+                  <li><a href="about.html">About</a></li>
+                  <li><a href="#program" >Program</a></li>
+                  <li><a href="#speak" >Speakers</a></li>
+                  <li><a href="#partners" >Partners</a></li>
+                  <li><a href="#footer" >Contact</a></li>
+                  <li><a href="/" class="header-list-conference">Conference</a></li>
+                </ul> 
+            </div>
+          </nav>
+          <div class="header-content">
+              <p>"Hello! Sharing world"</p>
+              <h1>BEIRUT GLOBAL<br>TECHNOLOGY CONFERENCE<br>2021</h1>
+              <div class="header-paragraph">
+                Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industrys standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it to make a type specimen book.
+              </div>
+              <h2>2021-12-09(THU) - 12(SUN) </h2>
+              <span>@ Biel Beirut waterfront, Art center</span>
+          </div>
```

Comment:
- [ ] ___[Optional]___ The hero section of your homepage is not well aligned when compared to that of the design template.

Original Design | Yours
----- | -----
![image](https://user-images.githubusercontent.com/16058470/128343619-5a293292-43d1-4e50-8829-30d27a2b703e.png) | ![image](https://user-images.githubusercontent.com/16058470/128343700-8cd20b19-c173-4d68-afea-a0e5f3bdcb47.png)

The design has the hero section aligned to the left as well as its content but your hero section is centred but has its content aligned to the left

Reply:
`This issue is well resolved! ✔️